components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider_finish.jsp

```diff
@@ -126,88 +126,70 @@
         }
 
         if (Boolean.parseBoolean(request.getParameter(SAMLSSOUIConstants.ENABLE_ATTRIBUTE_PROFILE))) {
-
             String claimsCountParameter = SAMLSSOUIUtil.getSafeInput(request, "claimPropertyCounter");
             if (claimsCountParameter != null && !"".equals(claimsCountParameter)) {
```

Comment:
Use IdentityUtil.isNotBlank(). This will check for StringUtils.isNotBlank() and also "null".